connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -166,6 +166,10 @@
     @Setter
```

Comment:
Since there were so many imports added to the other file there are likely some unused in this file.